config/initializers/warden.rb

```diff
@@ -8,8 +8,9 @@
 
 require 'warden/strategies/basic_strategy'
 require 'warden/strategies/session_strategy'
+require "warden/strategies/doorkeeper"
 
 Rails.application.config.middleware.insert_after(ActionDispatch::Flash, Warden::Manager) do |manager|
-  manager.default_strategies :basic, :session
+  manager.default_strategies :doorkeeper, :basic, :session
```

Comment:
just curious if we want doorkeeper to be used for all endpoints or only the api ... either way is fine, but just checking if you did that on purpose or by accident